_extras/snippets_library/EPCC_Cirrus_pbs/12/explore.snip

```diff
@@ -0,0 +1,10 @@
+> ## Explore a Worker Node
+> 
+> Finally, let's look at the resources available on the worker nodes where your jobs will actually
+> run. Try running this command to see the name, CPUs and memory available on the worker nodes:
+>
+> ```
+> {{ site.host_prompt }} pbsnodes r1i0n32
```

Comment:
Add a comment that the node name is provided by the instructor.

Reply:
Addressed in https://github.com/hpc-carpentry/hpc-intro/pull/101/commits/d61b43ba0fab708f940d9e87801e074ad95432db